core/trino-main/src/main/java/io/trino/sql/planner/iterative/rule/UnwrapCastInComparison.java

```diff
@@ -338,8 +339,8 @@ private Expression unwrapCast(ComparisonExpression expression)
                             yield new ComparisonExpression(LESS_THAN_OR_EQUAL, cast.getExpression(), literalEncoder.toExpression(session, literalInSourceType, sourceType));
                         }
                         case GREATER_THAN, GREATER_THAN_OR_EQUAL ->
-                                // We expect implicit coercions to be order-preserving, so the result of converting back from target -> source cannot produce a value
-                                // larger than the next value in the source type
+                            // We expect implicit coercions to be order-preserving, so the result of converting back from target -> source cannot produce a value
```

Comment:
@chenjian2664  why do you have now the extra commits?

Reply:
Strange ...
Let me check